addons/core/XEH_PREP.sqf

```diff
@@ -65,6 +65,8 @@ PREP(getDefaultRadioClasses);
 PREP(getRadioOwner);
 PREP(getSwSpeakers);
 PREP(getLrSpeakers);
+PREP(getSwHalfDuplexOverride);
```

Comment:
Fair enough, I was getting a bit confused by that as well. Will change it to SR.